packages/firebase_storage/firebase_storage/example/test_driver/task_e2e.dart

```diff
@@ -48,13 +48,16 @@ void runTaskTests() {
           await Future.delayed(Duration(milliseconds: 750));
         }
 
-        bool paused = await task.pause();
-        expect(paused, isTrue);
-        expect(task.snapshot.state, TaskState.paused);
-
-        bool resumed = await task.resume();
-        expect(resumed, isTrue);
-        expect(task.snapshot.state, TaskState.running);
+        // TODO(Salakar): Known issue with iOS where pausing/resuming doesn't immediately return as paused/resumed 'true'.
```

Comment:
If a good root cause comes from this would love a `@`-tag back as we have similar problem I think in RNFB https://github.com/invertase/react-native-firebase/blob/bda2d67ddaf7f12ac3143b564e7e94974e8356db/packages/storage/e2e/StorageTask.e2e.js#L628